config/ingest.py

```diff
@@ -1,17 +1,19 @@
 from lsst.obs.decam.ingest import DecamParseTask
+
 config.parse.retarget(DecamParseTask)
 config.parse.translation = {'visit': 'EXPNUM',
```

Comment:
I only reformatted the lines in ingestCuratedCalibs because they were >110chars. I preferred to not auto-format big blocks like this without getting broader feedback from the team about the use of `black` in configs, which don't always auto-format nicely.